app/pages/[teamSlug]/time/index.page.tsx

```diff
@@ -0,0 +1,71 @@
+import { format } from 'date-fns'
+import { useSession } from 'next-auth/react'
+import { useMemo, useState } from 'react'
+import { BiEdit, BiPlus, BiTimer } from 'react-icons/bi'
+import { BookWorkHourModal } from '../../../frontend/components/bookWorkHourModal'
+import { Button } from '../../../frontend/components/button/button'
+import { CalendarSelector } from '../../../frontend/components/calendarSelector'
+import { FormattedDuration } from '../../../frontend/components/duration/formattedDuration'
+import { ProtectedPage } from '../../../frontend/components/protectedPage'
+import { useWorkHoursQuery } from '../../../frontend/generated/graphql'
+
+const MaintainWorkHoursPage = () => {
+  const [isBookWorkHourModalOpen, setIsBookWorkHourModalOpen] = useState(false)
+  const [selectedDate, setSelectedDate] = useState(new Date())
+  const selectedDateString = format(selectedDate, 'yyyy-MM-dd')
+  const context = useMemo(() => ({ additionalTypenames: ['WorkHour'] }), [])
+  const [{ data }] = useWorkHoursQuery({ variables: { from: selectedDateString }, context })
+  return (
+    <ProtectedPage>
+      <BookWorkHourModal
+        selectedDate={selectedDate}
+        open={isBookWorkHourModalOpen}
+        onClose={() => setIsBookWorkHourModalOpen(false)}
+      />
+      <nav className="mt-5 mb-5 flex items-center justify-between">
+        <CalendarSelector onSelectedDateChange={setSelectedDate} />
+        <Button ariaLabel="add work item" variant="primary" onClick={() => setIsBookWorkHourModalOpen(true)}>
+          <BiPlus className="flex items-end text-3xl" />
+        </Button>
+      </nav>
+      <section className="mt-5 flex-row gap-2">
```

Comment:
This section is not a flex container. So `flex-row` and `gap-2` do not have any effect, right?